### PR TITLE
CTONET-530 Fix the issue from the podman change and also invoke the t…

### PR DESCRIPTION
**Pull request by jun-rh**
…est suite in run-tnf-container.sh

### CODE REVIEW

**Comment by jun-rh** [comment]
Forgot to update the image docker file for the script location change. Will update this.

**Comment by jun-rh** [comment]
@ryandgoulding @crwr45

File: run-cnf-suites.sh [hunk 1]
```diff
@@ -53,29 +53,11 @@ done
...
-	fi
-	
-	# For older verions of docker, dns server may need to be set explicitly, e.g.
-	#
-	# export DNS_ARG=172.0.0.53
-	./run-container.sh -ginkgo.v -ginkgo.skip="performance|sriov|ptp|sctp|xt_u32|dpdk|ovn"
-else
-	# removing report if not running, so the final claim won't include stale reports
-	rm -f ${OUTPUT_LOC}/validation_junit.xml
+if grep -q init\.scope /proc/1/cgroup; then
```
Comment (crwr45) [inline run-cnf-suites.sh:56]: This doesn't handle running on a Mac well:
```
cwheeler@MacBook-Pro test-network-function % TNF_ENABLE_CONFIG_AUTODISCOVER=true ./run-cnf-suites.sh generic
grep: /proc/1/cgroup: No such file or directory
```

**Review by crwr45 — CHANGES_REQUESTED** [review]
Looks good, one issue running on mac

File: run-cnf-suites.sh (hunk 1 above)
Comment (jun-rh) [reply run-cnf-suites.sh:56]: hmm, i can't test this as i don't have a mac. do you have this file when running in container then? i could do the opposite check. On linux, this file would contain something like "docker" or "user.slice".

Comment (jun-rh) [reply run-cnf-suites.sh:56]: @crwr45 @ryandgoulding Can one of you check the existence and the content of /proc/1/cgroup when running docker and podman on your MAC? Thanks.

Comment (jun-rh) [reply run-cnf-suites.sh:56]: Ok. I will use the absence of the file as another indicator of not running in container.

**Review by ryandgoulding — APPROVED** [review]

**Pull request merged**